core/src/storage_stage.rs

```diff
@@ -194,34 +196,40 @@ impl StorageStage {
         let exit1 = exit.clone();
         let keypair1 = keypair.clone();
         let storage_keypair1 = storage_keypair.clone();
+        let bank_forks1 = bank_forks.clone();
         let t_storage_create_accounts = Builder::new()
             .name("solana-storage-create-accounts".to_string())
-            .spawn(move || loop {
-                match tx_receiver.recv_timeout(Duration::from_secs(1)) {
-                    Ok(mut tx) => {
-                        if Self::send_transaction(
-                            &cluster_info0,
-                            &mut tx,
-                            &exit1,
-                            &keypair1,
-                            &storage_keypair1,
-                            Some(storage_keypair1.pubkey()),
-                        )
-                        .is_ok()
-                        {
-                            debug!("sent transaction: {:?}", tx);
+            .spawn(move || {
+                let transactions_socket = UdpSocket::bind("0.0.0.0:0").unwrap();
+                loop {
+                    match tx_receiver.recv_timeout(Duration::from_secs(1)) {
```

Comment:
why is there a thread responsible for moving stuff off a channel into the TPU?

Reply:
nevermind, I get it.

can you file an issue for "extra stage in storage thingie", and just send the TXes to the TPU directly instead?

Reply:
https://github.com/solana-labs/solana/issues/3986

Reply:
It was doing send/confirm/retry. So didn't want to wait around doing that in the main storage loop.